README.md

```diff
@@ -1,16 +1,9 @@
 # puppet-module-bind
 
-Manage bind9. This module's approach is to manage the configuration of bind9
-while not managing the data in the actual zones, which is up to you. Recommend
-keeping zone data in another repo and having a process sync that data to your
-bind servers.
-
-===
-
-### Table of Contents
-1. [Compatibility](#compatibility)
```

Comment:
This should probably be back and have entries for each class/define, since the README is going to be huge.